src/compas_ghpython/utilities/drawing.py

```diff
@@ -254,6 +255,16 @@ def draw_network(network):
 
     return points_rg, lines_rg
 
+def draw_circles(circles):
```

Comment:
i think if you add an additional newline here, all checks will pass :)

Reply:
right, I'll add it!